src/sentry/api/serializers/models/group.py

```diff
@@ -329,6 +329,8 @@ def serialize(
             "subscriptionDetails": subscription_details,
             "hasSeen": attrs["has_seen"],
             "annotations": attrs["annotations"],
+            "issueType": obj.issue_type.name.lower(),
+            "issueCategory": obj.issue_category.name.lower(),
```

Comment:
I wasn't sure if we wanted `group.Type`, `groupType`, `issue.Type`, etc. so I just went with this.